client/src/components/PortfolioTable.tsx

```diff
@@ -310,13 +310,30 @@ export const PortfolioTable = React.memo(
                   Object.entries(_groupBy(row.original.allcontacts, "value")).sort(
                     sortContactsByImportance
                   );
+
+                if (!contacts) return "";
+
+                // to ensure "+ #" contacts button sticks to last word in name with the rest wraping,
+                // they need to be in span together. 
+                var displayContactParts = contacts && contacts[0][0].match(/(.*)\s(.*)/);
+
                 return (
                   <>
-                    {contacts ? contacts[0][0] : ""}
-                    {row.getCanExpand() && contacts && contacts.length > 1 && (
-                      <button className="contacts-expand" onClick={row.getToggleExpandedHandler()}>
-                        +{contacts.length - 1}
-                      </button>
+                    {displayContactParts && displayContactParts[1] + " "}
+                    {row.getCanExpand() && contacts && contacts.length > 1 ? (
+                      <span className="col-ownernames-last-word">
+                        {displayContactParts ? displayContactParts[2] : contacts[0][0]}
+                        <button
+                          className="contacts-expand"
+                          onClick={row.getToggleExpandedHandler()}
+                        >
+                          +{contacts.length - 1}
+                        </button>
+                      </span>
+                    ) : displayContactParts ? (
+                      displayContactParts[2]
+                    ) : (
+                      contacts[0][0]
```

Comment:
yeah, sorry this whole thing is a bit of a mess, but can't think of how to simplify any further. 

If there is only one contact, with a 2-word name, `displayContactParts` will have the two words, so in this first line print the first name. Then the whole second chunk won't run because `contacts.length === 1`, so that won't add the second word and more-button. So then it will add just the second word at the step you're pointing out. And since the last `contacts[0][0]` only runs if `!displayContactParts` it won't run in this situation.

```
                    {displayContactParts && displayContactParts[1] + " "}
                    {row.getCanExpand() && contacts && contacts.length > 1 ? (
                      <span className="col-ownernames-last-word">
                        {displayContactParts ? displayContactParts[2] : contacts[0][0]}
                        <button
                          className="contacts-expand"
                          onClick={row.getToggleExpandedHandler()}
                        >
                          +{contacts.length - 1}
                        </button>
                      </span>
                    ) : displayContactParts ? (
                      displayContactParts[2]
                    ) : (
                      contacts[0][0]
                    )}
```

here's an example: 
![image](https://user-images.githubusercontent.com/16906516/222209845-0177f2eb-95cb-4ecb-8f0e-362402e0fa7d.png)

That last line is mean to handle a possible situation where there is a single contact, and it's a single word. (I assume that's rare, but you never know - maybe a single corporation name, typo, etc.). In this case the `displayContactParts` will be `null` from the regex match.

Reply:
hmm would it work to take the contact name -> trim -> split on space -> display each word in a span and append the plus button to the last element if it is expandable? might also be able to do something where you join everything that's NOT in the last word, but that might be more trouble than it's worth for only marginally cleaner dom structure...

Reply:
thanks so much! this makes so much more sense, and is way more simple/clear, than what I was doing before. 

Just updated with this fix. And I just did this to test out with the odd case of single word contact, and it works for that too. 
![image](https://user-images.githubusercontent.com/16906516/222221490-0a5c65bc-3993-49af-b548-31878b7e9a62.png)